NuKeeper/Engine/RepositoryUpdater.cs

```diff
@@ -69,6 +72,23 @@ public class RepositoryUpdater
             Console.WriteLine("Done");
         }
 
+        private async Task RestoreSolutions(IEnumerable<PackageUpdateSet> updates)
```

Comment:
This feels extractable into a separate class (`ISolutionManager.RestorePackages(update)`) which would make it more testable